cmd/worker/funcs_kmod.go

```diff
@@ -64,3 +59,56 @@ func kmodUnloadFunc(cmd *cobra.Command, args []string) error {
 
 	return w.UnloadKmod(cmd.Context(), cfg, mountPathFlag.Value.String())
 }
+
+func setCommandsFlags() {
+	kmodLoadCmd.Flags().String(
+		worker.FlagFirmwareClassPath,
+		"",
+		"if set, this value will be written to "+worker.FirmwareClassPathLocation,
+	)
+
+	kmodLoadCmd.Flags().String(
+		worker.FlagFirmwareMountPath,
+		"",
+		"if set, this the value that firmware host path is mounted to")
+
+	kmodLoadCmd.Flags().Bool(
+		"tarball",
+		false,
+		"if see, means run tarball",
+	)
+
+	kmodUnloadCmd.Flags().String(
+		worker.FlagFirmwareMountPath,
+		"",
+		"if set, this the value that firmware host path is mounted to")
+
+	kmodUnloadCmd.Flags().Bool(
+		"tarball",
+		false,
+		"if see, means run tarball",
+	)
+}
+
+func configureLogging() {
+	klogFlagSet := flag.NewFlagSet("klog", flag.ContinueOnError)
+	logConfig := textlogger.NewConfig()
+	logConfig.AddFlags(klogFlagSet)
+	logger = textlogger.NewLogger(logConfig).WithName("kmm-worker")
+	rootCmd.PersistentFlags().AddGoFlagSet(klogFlagSet)
+}
```

Comment:
Why do we need this as a separate function?

Reply:
just to make a code more cleaner